src/FluentValidation/Resources/Languages/EnglishLanguage.cs

```diff
@@ -46,6 +46,7 @@ internal class EnglishLanguage : Language {
 			Translate<EmptyValidator>("'{PropertyName}' must be empty.");
 			Translate<NullValidator>("'{PropertyName}' must be empty.");
 			Translate<EnumValidator>("'{PropertyName}' has a range of values which does not include '{PropertyValue}'.");
+			Translate<CountValidator>("'{PropertyName}' must have between {MinCount} and {MaxCount} elements. You provided {Count} elements.");

```

Comment:
Can we re-word this slightly? I'm not sure I like the "you provided" in here, as a collection length is not typically something that's directly user-specified (unlike say, the length of a string). How about:

```
'{PropertyName}' must have between {Min} and {Max} elements, but actually contains {Total} elements.
```

Reply:
Agree, done